scripts/jenkins/groovy/compareBenchmarksStage.groovy

```diff
@@ -322,7 +322,99 @@ def call(final pipelineContext, final stageConfig, final benchmarkFolderConfig)
                     'train_time_max': 5
                 ]
             ]
-        ]
+        ],
+        'rulefit': [
+            'redhat': [
+                [RULES, 3, 3]: [
```

Comment:
RULES is an undefined constant, so this will trigger a syntax error, needs to be a string

Reply:
fixed